packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -50,9 +52,19 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
                 return this.doOpen();
             }
         });
+        // Visible/enabled only on Windows/Linux in electron.
+        commands.registerCommand(WorkspaceCommands.OPEN_FILE, {
+            isEnabled: () => true,
+            execute: () => this.doOpenFile()
+        });
+        // Visible/enabled only on Windows/Linux in electron.
+        commands.registerCommand(WorkspaceCommands.OPEN_FOLDER, {
+            isEnabled: () => true,
+            execute: () => this.doOpenFolder()
+        });
```

Comment:
OPEN_FILE and OPEN_FOLDER are visible in electron pulldown menu under "File", but they are not visible from the command palette (CTRL+shift+p) and search for "ctrl+alt+o" (File->Open File...)

Reply:
That was my intention: https://github.com/theia-ide/theia/pull/3202/files#diff-7e70f039e1d4a1a1b544cf48997743e0R53